react/features/subtitles/components/Captions.web.js

```diff
@@ -76,7 +76,7 @@ class Captions
 function mapStateToProps(state) {
     return {
         ..._abstractMapStateToProps(state),
-        _isLifted: getParticipantCountWithFake(state) < 2
+        _isLifted: Boolean(getDominantSpeakerParticipant(state))
```

Comment:
Could you please explain why do we need this change? I'm having hard time to understand it.

Reply:
The old check was legacy, and deprecated. In the past we used to have a prompt at the bottom of the screen where people could open the AddPeopleDialog and subtitles needed to be lifted on top of it. That was removed but this was left behind, however now we will have a badge at the bottom displaying the dominant speaker name, so if there is a dominant speaker, we lift the subtitles.

Reply:
I see.

 I think that we show the dominant speaker badge in other cases too and we also hide it for the local participant. Isn't it? Shouldn't you match the conditions for showing the dominant speaker badge here?

Reply:
That's fair, I'll refactor it a bit.